pkgs/applications/graphics/rawtherapee/default.nix

```diff
@@ -20,17 +20,17 @@
 , librsvg
 , libcanberra-gtk3
 , gtk-mac-integration
+, exiv2
 }:
 
 stdenv.mkDerivation rec {
   pname = "rawtherapee";
-  version = "5.9";
+  version = "5.10";
 
-  src = fetchFromGitHub {
-    owner = "Beep6581";
-    repo = "RawTherapee";
+  src = fetchgit {
```

Comment:
Why switch from fetchFromGitHub

Reply:
In the [release notes](https://www.rawtherapee.com/downloads/5.10/#news-relevant-to-package-maintainers) from RawTherapee's maintainers they state this :

> To get the source code, either clone from git or use the tarball from https://rawtherapee.com/shared/source/ . Do not use the auto-generated GitHub release tarballs.

I'm not sure how this could differ though, but if it can it could prevent some subtle issue in the future? :shrug:

Reply:
I see

Reply:
Should be fine

Reply:
Would `forceFetchGit = true` do the job?

Reply:
Yes, it works :+1: